### ISSUE TITLE
No templates or statics in tarball

### ISSUE THREAD

**Issue by ergoithz** [ergoithz/browsepy#1]

**Comment by ergoithz** [comment]
```
jinja2.exceptions.TemplateNotFound: 404.html
Traceback (most recent call last):
  File "/home/music/app/env/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/music/app/env/lib/python3.4/site-packages/flask/app.py", line 1479, in full_dispatch_request
    response = self.process_response(response)
  File "/home/music/app/env/lib/python3.4/site-packages/flask/app.py", line 1691, in process_response
    response = handler(response)
  File "/home/music/app/env/lib/python3.4/site-packages/browsepy/__init__.py", line 455, in page_not_found
    return make_response((render_template('404.html'), 404))
  File "/home/music/app/env/lib/python3.4/site-packages/flask/templating.py", line 127, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/music/app/env/lib/python3.4/site-packages/jinja2/environment.py", line 851, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/music/app/env/lib/python3.4/site-packages/jinja2/environment.py", line 812, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/music/app/env/lib/python3.4/site-packages/jinja2/environment.py", line 774, in _load_template
    cache_key = self.loader.get_source(self, name)[1]
  File "/home/music/app/env/lib/python3.4/site-packages/flask/templating.py", line 64, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: 404.html
```

When mounting under cherrypy as
```
#!/home/music/app/env/bin/python

import cmbootstrap
cmbootstrap.bootstrap()
import cherrymusicserver
import cherrypy

from os.path import expandvars
from browsepy import app as browsepy

cherrymusic_config = {
    'server.rootpath': '/player'
}

browsepy.config.update(
    APPLICATION_ROOT = '/browse',
    directory_base = expandvars('$home/media'),
    directory_start = expandvars('$home/media'),
    directory_remove = expandvars('$home/media'),
    directory_upload = expandvars('$home/media'),
)

if __name__ == '__main__':
    cherrymusicserver.setup_config(cherrymusic_config)
    cherrymusicserver.setup_services()
    cherrymusicserver.migrate_databases()
    cherrypy.tree.graft(browsepy, '/browse')
    cherrymusicserver.start_server(cherrymusic_config)
```

**Comment by ergoithz** [comment]
Fixed on master and [0.3.8 release](https://github.com/ergoithz/browsepy/releases/tag/0.3.8) ([0.3.8.3 on pypi](https://pypi.python.org/pypi/browsepy/0.3.8)).
